cmd/pulumi-test-language/interface.go

```diff
@@ -262,8 +265,10 @@ func compareDirectories(actualDir, expectedDir string, allowNewFiles bool) ([]st
 		}
 
 		if !bytes.Equal(actualContents, expectedContents) {
-			// TODO(https://github.com/pulumi/pulumi/issues/13943): Find a way to show better diffs here
-			validations = append(validations, fmt.Sprintf("expected file %s does not match actual file", relativePath))
+			edits := myers.ComputeEdits(span.URIFromPath("expected"), string(expectedContents), string(actualContents))
```

Comment:
This seems unrelated. Can you remove the changes in this file and do it as a separate change.?

Reply:
👍 https://github.com/pulumi/pulumi/pull/14786